private/buf/bufcli/bufcli.go

```diff
@@ -549,19 +556,22 @@ func newModuleReaderAndCreateCacheDirs(
 	cacheModuleDataDirPath := normalpath.Join(container.CacheDirPath(), v1CacheModuleDataRelDirPath)
 	cacheModuleLockDirPath := normalpath.Join(container.CacheDirPath(), v1CacheModuleLockRelDirPath)
 	cacheModuleSumDirPath := normalpath.Join(container.CacheDirPath(), v1CacheModuleSumRelDirPath)
+	cacheModuleDirPathV2 := normalpath.Join(container.CacheDirPath(), v2CacheModuleRelDirPath)
 	if err := checkExistingCacheDirs(
```

Comment:
Just a question - in theory, should a new version of `buf` need to create the v1 cache dirs at all? Will it once tamper proofing is everywhere?

Reply:
Nope - we'd only need the v2 directories. I can update to create only the necessary directories.